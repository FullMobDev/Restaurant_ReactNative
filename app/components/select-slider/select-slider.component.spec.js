import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow } from 'enzyme';
import React from 'react';

import SelectSlider from '../select-slider/select-slider.component';

Enzyme.configure({ adapter: new Adapter() });

describe('<SelectSlider />', () => {
    // test('displays a list of <SelectSliderItem> from options passed in', () => {
    //     const options = ['option1', 'option2'];
    //     const wrapper = shallow(<SelectSlider options={options} />);

    //     expect(wrapper).toMatchSnapshot();
    // });
});
