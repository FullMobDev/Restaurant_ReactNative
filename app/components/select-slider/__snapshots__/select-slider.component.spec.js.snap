// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectSlider /> displays a list of <SelectSliderItem> from options passed in 1`] = `
<View>
  <SelectSliderOption
    option="option1"
    selectPositionX={0}
    selected={false}
  />
  <SelectSliderOption
    option="option2"
    selectPositionX={0}
    selected={false}
  />
</View>
`;

exports[`<SelectSlider /> takes as input a list of items 1`] = `<View />`;
